Moodis/Moodis/cameraForm.cs

```diff
@@ -0,0 +1,100 @@
+﻿using AForge.Video;
+using AForge.Video.DirectShow;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace moodie
+{
+    public partial class cameraForm : Form
+    {
+        public cameraForm()
+        {
+            InitializeComponent();
+        }
+
+        private FilterInfoCollection webcam;
+        private VideoCaptureDevice cam;
+        private const string WarningMsg = "You must first turn on the camera!";
+
+        private void CameraFormLoad(object sender, EventArgs e)
+        {
+            webcam = new FilterInfoCollection(FilterCategory.VideoInputDevice);
+            foreach(FilterInfo VideoCaptureDevice in webcam)
```

Comment:
change FilterInfo to var

Reply:
I don't believe he can change that based on the documentation www.aforgenet.com/framework/docs/html/8d13e608-64f7-8c03-84b1-1bce74dabecd.htm